### PR TITLE
Bumped scipp to 23.08

### PR DESCRIPTION
**Pull request by andped10**
New version of scipp uses plopp as default plotting tool.

plot.py
- plopp doesn't allow for variance in coords data.
- changed to one argument per line (easier to see differences in git)
- removed try / expect to handle code logic

In pyproject.toml:
- bumped scipp to 23.08
- dev dependencies
  - alphabetic order
  - added plopp

Adjusted jupyter notebooks to new plotting functionality.

Adjust read in data to the more strict requirements by scipp (removed plural s in example data)

### CODE REVIEW

File: docs/_static/d70d2o.ort [hunk 1]
```diff
@@ -9,7 +9,7 @@
 #         start_date: 2015-05-20
 #         title: A single contrast of DSPC
 #         instrument: surf
-#         probe: neutrons
+#         probe: neutron
```
Comment (rozyczko) [inline docs/_static/d70d2o.ort:12]: we probably shouldn't modify the database-retrieved files. Is there any reason to singularise `neutrons` and `x-rays`?

File: docs/solvation.ipynb [hunk 2]
```diff
@@ -46,10 +56,21 @@
...
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "numpy: 1.26.3\n",
+      "scipp: 23.08.0\n",
+      "EasyReflectometry: 0.0.2\n",
+      "refnx: 0.1.41\n"
+     ]
+    }
+   ],
```
Comment (rozyczko) [inline docs/solvation.ipynb:73]: all these extra outputs mean that you probably didn't clean/reset the notebook before committing. This should be done so the initial state is an empty notebook.

**Review by rozyczko — CHANGES_REQUESTED** [review]
Looks good. Two minor issues raised, once resolved the branch can be merged.

File: docs/_static/d70d2o.ort (hunk 1 above)
Comment (andped10) [reply docs/_static/d70d2o.ort]: The reason was that orsopy now throws a warning:
image

I have reverted the changes.

File: docs/solvation.ipynb (hunk 2 above)
Comment (andped10) [reply docs/solvation.ipynb:73]: No, they are intended.  It is a clean notebook.  The reason for the new "code" parts is that I added a new markdown section stating that we setup the matplotlib to plot the figures inline and import the needed modules:

```
%matplotlib inline

import ...
```

Comment (andped10) [reply docs/solvation.ipynb]: Had misunderstood PR comment:

Removed output form solvation.ipynb

File: docs/_static/d70d2o.ort (hunk 1 above)
Comment (rozyczko) [reply docs/_static/d70d2o.ort]: Good to know. Existing orsopy dependency is >=0.04, while current orsopy is 1.1.0. Probably this is why.
At some point, we will want to update the data files.

**Review by rozyczko — APPROVED** [review]
looks good!

**Pull request merged**